packages/logging/test/entry.js

```diff
@@ -44,8 +44,23 @@ describe('Entry', function() {
   });
 
   describe('instantiation', function() {
-    it('should localize metadata and data', function() {
-      assert.strictEqual(entry.metadata, METADATA);
+    it('should assign timestamp to metadata', function() {
+      assert.deepEqual(entry.metadata, extend({
+        timestamp: new Date()
```

Comment:
I suspect this might spuriously fail depending on timing / machine load. The new Date object may not equal the date captured by the constructor a few statements ago.